docs/banzai_nres/index.rst

```diff
@@ -8,3 +8,135 @@ Reference/API
+detector, whose calculated wavelength under our wavelength solution model is close (in wavelength) to a known laboratory
+line. Close is currently defined as 0.1 Angstroms.
+
+Now some more detail on each.
+
+'MLINENUM' is the number of matched lines.
+
+'LINENUM' is the total number of lines observed on the detector.
+
+The last three quantities are calculated from a single distribution, after applying various transformations to it.
+This distribution is calculated as follows: for the wavelength
+of each feature observed on the detector,
+find the nearest line in the laboratory line list. Calculate that difference. Record those differences. Now restrict
+the resulting distribution only to matched lines. This is the
+distribution, which we will refer to as the distribution of residuals. Again, note that this distribution is restricted
+to matched lines only.
+
+
+'SIGLAM' is the standard deviation of the distribution of residuals.
+
+'PRECISN' is slightly more complicated. It is the error on the estimate of the mean of the distribution of residuals,
+converted
+into velocity via delta lambda / lambda = v/c . I.e. it is how well you know zero-point of the wavelength residuals
+(in velocity space). This naively sets the maximum precision you can attain on the instrument.
+
+'CHISQ' this is the formal chisquared statistic of the distribution of residuals, i.e. each residual divided by the
+standard error (in wavelength) of the line centroid position (in wavelength).
+
+
+What is a good wavelength solution?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are a variety of checks to make sure your wavelength solution is good. Most of them involve simply checking
+the numbers from the above section.
+
+A good wavelength solution for any NRES unit will have those quantities in the following ranges:
+
+
+* | 'SIGLAM' This should be between 0.003 and 0.01 Angstroms. It should be roughly equal to
+  | 'PRECISN' / speed of light * sqrt('MLINENUM') * (mean wavelength). mean wavelength = 5000 Angstroms is fine.
+* | 'PRECISN' This should be between 4 and 10 m/s . High m/s values above 20 might indicate a failure of the wavelength
+  | solution, or extremely low S/N on the frame.
+* | 'CHISQ' between 0.2 and 5. An extremely high number here might mean errors are massively underestimated and
+  | vice-versa.
+* | 'LINENUM' between 3500 and 5000 (i.e. 1750 and 2500 lines per fiber). The code as is limits this value to 2500
+  | features per fiber.
+* | 'MLINENUM' should be roughly 50-80% of LINENUM, usually 70%. E.g. 3500 for 5000 total lines, or 2450 for 3500 total
+  | lines. If 'MLINENUM' is below 1000, your wavelength solution is probably overconstrained in the center of the
+  | detector and the edges are missed. This frame is probably too low of signal to noise.
+
+
+If all the above checks pass, you most likely have a very good wavelength solution. If any fail, you may want to dig
```

Comment:
While probably true, "digging into the wavelength solution" is too generic. If we want to give advice we need to have concrete things to test.